git/project_git_remotes.md

```diff
@@ -1,5 +1,5 @@
 # Project: Git Remotes
-A brief introduction to the project and an over view of what the student will be building.
+In this project we will be forking TheOdinProject cirriculum and learning to add a solution to the cirriculum through a pull request. This is a common way to contribute to open source software on GitHub and valuable skill. Your pull request will be reviewed by the team at TheOdinProject and once approved will be displayed on this page.
```

Comment:
I don't think we will be displaying their solution on this page. We will be creating a separate repo in the Odin Organization for this project. It will just contain a text file where students will add their name to to the list.